ai_council/main.py

```
@@ -74,6 +70,46 @@ def __init__(self, config_path: Optional[Path] = None):
+            execution_mode
+        )
+        
+        try:
+            response = future.result(timeout=timeout_seconds)
+            
+            if response.success:
+                self.logger.info("Request processed successfully")
+            else:
+                self.logger.warning(f"Request processing failed: {response.error_message}")
+            
+            return response
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)